builder/__main__.py

```diff
@@ -38,7 +37,7 @@
 ########################################################################################################################
 
 def run_action(action, env):
-
+    config = env.config
     # Set build environment from config
     env.shell.pushenv()
     for var, value in config.get('build_env', {}).items():
```

Comment:
This was crashing before, right?

It wasn't picking up some other `config` defined somewhere else? If this was the case, let's change a variable name somewhere so this mistake isn't so easy to make.

but probably this was just a crash

Reply:
Yeah, it was picking up the one at global scope from __main__.  I got rid of that.